packages/dds/map/src/mapKernel.ts

```diff
@@ -41,59 +47,14 @@ interface IMapMessageHandler {
      */
     submit(op: IMapOperation, localOpMetadata: unknown): void;
```

Comment:
Should the `submit` signature use the same metadata typing as `applyStashedOp`?

Reply:
not sure, @scarlettjlee  ?

Reply:
For consistency here, it seems like it should. I considered updating the type for all the localOpMetadata in IMapMessageHandler when I was making the rollback changes earlier, but it resulted in duplicated casting and checking in multiple methods in the same callstack that seemed unnecessary. You could also go the opposite way and make the return type of IMapMessageHandler unknown again.

Reply:
I pushed an approach to cast metadata types. It doesn't look bad imo